extension/lib/commands.coffee

```diff
@@ -16,6 +16,7 @@ translate = require('./translate')
 utils = require('./utils')
 viewportUtils = require('./viewport')
 
+gDevToolsBrowser = Cu.import('resource://devtools/client/framework/gDevTools.jsm').browser
```

Comment:
Does this work in older Firefox as well?

Reply:
Yes. That file has been around for quite a while.

Reply:
I started the release process, but got a warning on AMO.

> Warning: The gDevTools.jsm module should no longer be used. The object should now be loaded using require("devtools/client/framework/devtools"). See https://bugzil.la/1188405 for more information.

I've paused the release for now. Is this something you could look into?

Reply:
Yeah, I noticed this when I was developing (`gDevTools.jsm` has a comment saying it is a compatibility shim)  but `require(...)` failed because when the `devtools/client/framework/devtools` module tried to `require` its dependencies it failed because the path was messed up (something like `resource://vimfx/foo/bar/dependecy` IIRC).

It would be really good to figure out how to get `require` working as this seems to be the future however I couldn't figure it out. I'll try to investigate further this weekend to see if I can crack it.

Reply:
Note that VimFx has its own `require` function, and hasn't got access to the `require` function sometimes seen in some parts of Firefox's source code.

Reply:
I added another commit which loads it as requested however it appears to be flakey for some reason. No errors are shown.

Reply:
Moving to https://github.com/akhodakivskiy/VimFx/pull/924